packages/labs/gen-manifest/src/index.ts

```diff
@@ -114,12 +113,18 @@ const convertModule = (module: Module): cem.Module => {
   };
 };
 
+const convertCommonInfo = (info: NodeJSDocInfo) => {
+  return {
+    ...pickIfNotEmpty(info, 'description'),
+    ...pickIfNotEmpty(info, 'summary'),
+    ...pickIfNotEmpty(info, 'deprecated'),
+  };
```

Comment:
`undefined` properties aren't written out with `JSON.stringify()`, so might it be nicer to not use spread to write a single value, and do something like:

```suggestion
  return {
    description: removeEmpty(info.description),
    summary: removeEmpty(info.summary),
    deprecated: removeEmpty(info.deprecated),
  };
```

Or, are the all the properties in NodeJSDocInfo always going to be used? Could you transform them all?

```suggestion
  return removeEmpties(info);
```

Reply:
> undefined properties aren't written out with JSON.stringify()

OMG! 🤦  Yeah I think I knew this but totally forgot about it... definitely better to leave the key and avoid the spread. Will refactor.

Reply:
Refactored into
* `ifNotEmpty(v)
* `transformIfNotEmpty(v, transform)
which always return a meaningful value or undefined, and are assigned directly to keys (not spread)